vendor/mozilla-rust-sdk/Cargo.toml

```diff
@@ -3,3 +3,5 @@ members = [
   "googleapis",
   "googleapis-raw",
 ]
+
+# ci kick -- can be deleted
```

Comment:
There should be a restart build button on Travis and Circle.

Reply:
There is, but for some stupid reason, I don't believe that Circle got the webhook notice that anything was pending. I looked on the circle task list and there was nothing. 

Fun of complex, interwoven systems. Lots of room for stuff to break in fun ways.

Reply:
There is also a way to tell GitHub to resend the webhook, if you have access to the repo settings.